### PR TITLE
PD-21: add price endpoint

### PR DESCRIPTION
**Pull request by fedealconada**

### CODE REVIEW

File: src/contracts/tokenDB.json [hunk 1]
```diff
@@ -1,16 +1,28 @@
 [
   {
-    "constant": true,
-    "inputs": [],
-    "name": "getTokens",
-    "outputs": [
+    "constant": false,
```
[the remark this answers is not in the document]
Comment (fedealconada) [reply src/contracts/tokenDB.json]: i think it was just that the tokenDB.json that we had was and old one... cause I realised that there were some functions not present on the file

File: src/prices/controller.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * @param {object} res - respond object.
+ * @return {json} The result.
+ */
+const getPrice = async (req, res) => {
+  const { from, to = 'USD' } = req.query
+  const api =`${CRYPTO_COMPARE}/price?extraParams=qiibee&fsym=ETH&tsyms=${to}`
+  const { status, data } = await axios.get(api)
+  const rate = await tokenRate(from)
+
+  let statusCode = 200
```
Comment (danoctavian) [inline src/prices/controller.ts]: we can use HttpStatus.OK for these constants

File: src/prices/controller.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  let results = {}
+
+  if (status !== 200 || data.Response) {
+    statusCode = data.Response ? 400 : status
```
Comment (danoctavian) [inline src/prices/controller.ts]: HttpStatus.BAD_REQUEST = 400

File: __tests__/integration/prices.test.ts [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+      .get(`/prices`)
+      .query(pricesParams)
+
+    expect(response.status).toBe(HttpStatus.OK)
+    expect(response.body).toEqual({
+      "USD": ((ETH_PRICE_QBX * ETH_PRICE_USD) / TOKEN.rate).toFixed(4),
+    })
+  })
+
+  it('Gets price of LoyaltyToken MCW should not fail if currency unknown', async () => {
```
Comment (danoctavian) [inline __tests__/integration/prices.test.ts]: i guess here the wording is `should fail` ? since our endpoint returns a 400

**Pull request merged**